packages/rest/src/rest.server.ts

```diff
@@ -236,11 +237,28 @@ export class RestServer extends Context implements Server, HttpServerLike {
     });
 
     // Mount our error handler
-    this._expressApp.use(
-      (err: Error, req: Request, res: Response, next: Function) => {
-        this._onUnhandledError(req, res, err);
-      },
-    );
+    this._expressApp.use(this._unexpectedErrorHandler());
+  }
+
+  /**
+   * Get an Express handler for unexpected errors
+   */
+  protected _unexpectedErrorHandler(): ErrorRequestHandler {
+    return (err, req, res, next) => {
```

Comment:
Please use a named function, so that if any code inside this handler throws an error, we have a useful stack frame in the error stack trace. That's why the original code uses `=>` function when calling `app.use` and then deferring to `this._onUnhandledError`.

```suggestion
    return function handleUnexpectedError(err, req, res, next) {
```

Reply:
`const functionName = (arg) => {...}` is a better way to name an arrow function. See https://stackoverflow.com/questions/27977525/how-do-i-write-a-named-arrow-function-in-es2015/37488652.